internal/pkg/template/templates/environment/cf.yml

```
@@ -100,6 +103,14 @@ Resources:
     Type: AWS::EC2::SecurityGroup
     Properties:
       GroupDescription: Access to the public facing load balancer
+       # Change load balancer to only accept ingress from CloudFront when prefix list specified
+{{- if .CDNConfig.PrefixListID}}
+      SecurityGroupIngress:
```

Should this be a list? https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-ec2-security-group.html#cfn-ec2-securitygroup-securitygroupingress
```suggestion
      SecurityGroupIngress:
        - SourcePrefixListId:
```

---

For this and the below comment, it's relevant to the prefix list, so I'll address it in the next PR, and am removing these changes from the current one to reduce the scope. See comment below that as well.